src/variables/index.ts

```diff
@@ -98,67 +120,81 @@ export namespace Variables {
     }
 
     private _filterVariables(): IVariable[] {
-      return this._state.filter(
+      return this._currentScope.variables.filter(
         ele =>
           ele.name
             .toLocaleLowerCase()
             .indexOf(this._filterState.toLocaleLowerCase()) !== -1
       );
     }
 
-    private _current: IVariable;
+    private _currentVariable: IVariable;
     private _currentChanged = new Signal<this, IVariable>(this);
     private _variablesChanged = new Signal<this, IVariable[]>(this);
     private _filterState: string = '';
-    private _state: IVariable[];
+    protected _state: IScope[];
+    private _currentScope: IScope;
   }
 
   export interface IOptions extends Panel.IOptions {}
 }
 
 const MOCK_DATA_ROW = {
-  variables: [
-    {
-      name: 'test 1',
-      value: 'function()',
-      type: 'function',
-      variablesReference: 0,
-      description: 'def test1(): return 0'
-    },
-    {
-      name: 'Classtest',
-      value: 'class',
-      type: 'class',
-      variablesReference: 1,
-      description: 'def test2(): return 0'
-    },
-    {
-      name: 'test 3',
-      value: 'function()',
-      type: 'function',
-      variablesReference: 0,
-      description: 'def test1(): return 0'
-    },
+  scope: [
```

Comment:
How about using the plural form?

```suggestion
  scopes: [
```

Reply:
Yeah, it's obvious. My bad

Reply:
All good :+1: